### ISSUE TITLE
Why doesn't the loss change very much after initialization?

### ISSUE THREAD

**Issue by JialeChe** [ptran1203/pytorch-animeGAN#25]
In training, the loss of the discriminator decreases, but the loss of the generator increases
![image](https://github.com/ptran1203/pytorch-animeGAN/assets/147423760/466c87d1-5a81-462b-815b-831f89acbe43)

**Comment by ptran1203** [comment]
It maybe the Discriminator is stronger than Generator, maybe because learning rate of D is twice higher than G. You can adjust learning rate each of them.

Beside of that, you can check if generated images look good or not. It's easier to understand indicator than loss.
